templates/service.yaml

```diff
@@ -1,6 +1,10 @@
 apiVersion: v1
 kind: Service
 metadata:
+{{- if .Values.service.annotations }}
+  annotations:
+    {{- toYaml .Values.metallb.annotations . | nindent 4 }}
```

Comment:
should this point to `.Values.service.annotations` instead of `.Values.metallb.annotations`?

Reply:
yes it should! fixed